tests/integration/conftest.py

```diff
@@ -214,9 +212,11 @@ def datarow(dataset, image_url):
 @pytest.fixture
 def label_pack(project, rand_gen, image_url):
     client = project.client
-    dataset = client.create_dataset(name=rand_gen(str), projects=project)
-    data_row = dataset.create_data_row(row_data=image_url)
+    dataset = client.create_dataset(name=rand_gen(str))
+    project.datasets.connect(dataset)
+    data_row = dataset.create_data_row(row_data=IMG_URL)
     label = project.create_label(data_row=data_row, label=rand_gen(str))
+    time.sleep(10)
```

Comment:
That seems like a lot of time. Is there any way to avoid this? I assume that things were not being created in time? is there a way to "await" (for lack of a pythonic verison of the word) these function calls?

Reply:
everything is technically awaited in python. The problem is the async stuff on the backend that can't be awaited.

Reply:
Also once label imports are complete, I can throw out this garbage hack to upload labels for tests.

Reply:
ah, yeah. dang. 10s is a while to add to a test, but if the processing takes that long, makes sense!